src/explore_tools.py

```diff
@@ -0,0 +1,58 @@
+
+from os import listdir
+from os.path import join, isfile, isdir
+
+import config
+from models.config import Config
+
+def glob_all(namespace):
```

Comment:
This is particularly descriptive, possible `glob_config` would be better?

Reply:
Done here https://github.com/alces-software/adminware/pull/99/commits/d8d2955436d53392d83c52ce181865bb36cf5878

Reply:
I think `glob_configs` is sufficient here instead of `glob_all_configs`

Reply:
The word 'config(s)' is positively abused throughout this codebase (seriously grep `src` for config) PLUS every function in that module could be reasonably described as 'glob configs' - I feel that the 'all' is really needed for specificity